.github/workflows/e2e-container.yaml

```diff
@@ -0,0 +1,37 @@
+name: End to end tests - containerd
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+
+      - run: |
+          pip3 install --user --upgrade j2cli
+          j2 --version
+
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+
+      - name: Setup kind cluster
+        run: hack/e2e-kind-cluster-setup.sh
+
+      - name: Provision the examples into the cluster
```

Comment:
maybe worth to mention on the name that it is a test, not just provision

Reply:
Ok, I'll try to give a more accurate explanation of what this is.